frontend/scss/components/organisms/header.scss

```diff
@@ -56,6 +59,18 @@
     padding-left: 30px;
   }
 
+  input {
+    display: none;
+  }
+
+  input[name=ap-o-header-main-item-radio-btn]:checked ~ .ap-o-header-flyout {
```

Comment:
That selector looks dirty. Also you are using something that looks like a class selector for the name attribute. I'd favour an selector like `.ap-o-header-main-item-radio-btn:checked`.

Reply:
since we discussed the issue with the second level flyout radio button being nested inside the link tag for each primary list item, I've removed the second radio button and it's input.